src/zenml/integrations/gcp/orchestrators/vertex_orchestrator.py

```diff
@@ -414,6 +438,23 @@ def _construct_kfp_pipeline() -> None:
                     ]
                     container_op.after(upstream_container_op)
 
+                # Set optional CPU, RAM and GPU constraints for the pipeline
+                if self.cpu_limit is not None:
+                    container_op = container_op.set_cpu_limit(self.cpu_limit)
+
+                if self.memory_limit is not None:
+                    container_op = container_op.set_memory_limit(
+                        self.memory_limit
+                    )
+
+                if self.node_selector_constraint is not None:
+                    container_op = container_op.add_node_selector_constraint(
+                        self.node_selector_constraint
+                    )
```

Comment:
Not sure which version of kfp you were running with, for me when trying this out in my dev-environment i ran into issues with the .add_node_selector_constraint as the second positional argument is missing, would this work for you:

```suggestion
                    container_op = container_op.add_node_selector_constraint(
                        label_name="cloud.google.com/gke-accelerator",
                        value=self.node_selector_constraint
                    )
```

If I understood the docs correctly, this "cloud.google.com/gke-accelerator" is currently the only available label name

Reply:
Changed the node selector to contain the label name. See above.